paddle/fluid/framework/ir/fused_multi_transformer_decoder_pass_tester.cc

```diff
@@ -474,9 +474,9 @@ TEST(MultiDevicesFusedMultiTransformerDecoderFuseQKVPass, basic) {
   auto* bias_l = layers.data("weightsl", {1024, 1024}, true);
   auto* linear_matmut_out =
       layers.matmul_v2(reshape_qkv_out, weights_l, nullptr, false, true);
-  auto* c_allreduce_out = layers.c_allreduce_sum(linear_matmut_out);
+  auto* all_reduce_out = layers.all_reduce(linear_matmut_out);
```

Comment:
all_reduce_out -> allreduce_sum_out

Reply:
Thx, the variable names has been modified.